### ISSUE TITLE
Improved screen reader support

### ISSUE THREAD

**Issue by StormProductionsMusic** [rustdesk/rustdesk#2536]
Hi,

Am using a screen reader (called NVDA) and am unable to read the password at all. 

When you want to see the password, you have to hover the mouse over it.  Unfortunately, for my case, there's no way I can do this, and so I have to call sighted assistance in order to give the password to my friend.

Also, a lot of buttons are unlabelled (screen reader can't see them), so I can not, for example, accept incoming connections from other people (to avoid giving out the password).

So, to make it short, my suggestion is to improve the screen reader support. I can definitely help testing that, so let me know if my help is needed.

Thanks for reading. Have a nice time! :)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/1101
